app/controllers/payment/connection/gemini_connections_controller.rb

```diff
@@ -58,6 +58,7 @@ def destroy
 
         # Destroy our database records
         if gemini_connection.destroy
+          current_publisher.update(wallet_provider: nil)
```

Comment:
I think you need to rebase to use `selected_wallet_provider` and do the ID check

Reply:
https://github.com/brave-intl/publishers/pull/2859/commits/0fc4bcc81bf7fe6b9de1e906b9307acb87b8590b